endpoint/BUILD

```diff
@@ -1,3 +1,5 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Which else would use the bazel rules under this BUILD file

Reply:
Test files inside unit test.